keyboards/gopolar/gg86/keymaps/horrortroll/oled/glcdfont.c

```diff
@@ -0,0 +1,232 @@
+// This is the 'classic' fixed-space bitmap font for Adafruit_GFX since 1.0.
+// See gfxfont.h for newer custom bitmap font info.
+
+#include "progmem.h"
+
+// Standard ASCII 5x7 font
+const unsigned char font[] PROGMEM = {
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
ooh, nice font. where is it from?

Reply:
It is Dead Meat font, and some custom symbol i’ve make for keylogger function

Reply:
https://www.whatfontis.com/NMY_Dead-Meal.font
Sorry, It this one. I’ve been remember mistake last name

Reply:
If it's a purchased font with a commercial license, then it's problematic at best going into the repo, as the license for the font may not permit that.

Reply:
Well huh, but i've only use it only for my keymap. So, well i can just revert it back.

Reply:
Just reverted back to default font